manifests/2.12.0/opensearch-2.12.0.yml

```diff
@@ -56,6 +56,8 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+    depends_on:
+      - job-scheduler
```

Comment:
Where is the code that process the `depends_on`?

Reply:
Mainly it's here.
https://github.com/opensearch-project/opensearch-build/blob/fec0355ffdd353e6885d76cf997d9c81cd723507/src/build_workflow/build_incremental.py#L60-L61